moped-editor/src/views/staff/StaffListView.js

```diff
@@ -90,8 +90,8 @@ const staffColumns = [
   {
     headerName: "Last seen",
     field: "last_seen_date",
-    valueGetter: (props) =>
-      props.value ? new Date(props.value).toLocaleString() : "",
+    type: "date",
```

Comment:
Will anyone miss seeing this down to the second?! 👀 👀 👀 😆 100% being sarcastic, and I like the cleaner look!

Reply:
i need to know *to the second* who is beating me when signing in to moped

Reply:
i'm you pointed this out @mddilley — because i didn't even notice this change. as someone who obsessively watches Moped activity, I will actually kinda miss the timestamps 😥. but i agree cleaner is better 🙏